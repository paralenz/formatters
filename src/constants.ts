export const BAR_TO_PSI_RATIO = 14.5037738
export const METERS_TO_FEET_RATIO = 3.28084
export const LBS_TO_KG_RATIO = 2.20462262185
export const KELVIN = 273.15
export const ONE_BYTE = 1024.0
