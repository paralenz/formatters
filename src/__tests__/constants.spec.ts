import * as C from '../constants'

describe('constants', () => {
  it('should match snpashot', () => {
    expect(C).toMatchSnapshot()
  })
  })