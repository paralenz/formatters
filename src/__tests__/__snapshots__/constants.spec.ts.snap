// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants should match snpashot 1`] = `
Object {
  "BAR_TO_PSI_RATIO": 14.5037738,
  "KELVIN": 273.15,
  "LBS_TO_KG_RATIO": 2.20462262185,
  "METERS_TO_FEET_RATIO": 3.28084,
  "ONE_KILOBYTE": 1024,
  "PRESSURE": Object {
    "bar": 1,
    "psi": 0.0689475729,
  },
}
`;
